crypto/bn/asm/rsaz-avx2.pl

```diff
@@ -1815,14 +1857,17 @@
 	cmp	%r10,%rbx		# context->Rip<prologue label
 	jb	.Lcommon_seh_tail
 
-	mov	152($context),%rax	# pull context->Rsp
-
 	mov	4(%r11),%r10d		# HandlerData[1]
 	lea	(%rsi,%r10),%r10	# epilogue label
 	cmp	%r10,%rbx		# context->Rip>=epilogue label
 	jae	.Lcommon_seh_tail
 
-	mov	160($context),%rax	# pull context->Rbp
+	mov	160($context),%rbp	# pull context->Rbp
```

Comment:
Right, it's unrelated. It's something I simply ran across and fixed in passing. I'll split it into separate commit. It probably should even be a separate request...